web/styles/recent_view.css

```diff
@@ -4,548 +4,528 @@
-                    min-width: 7px;
-                    height: 7px;
-                    float: left;
-                    position: unset;
-                }
-            }
+        .recent_topic_actions {
+            margin-right: 5px;
+            font-size: 15px;
         }
     }
 
-    #recent_view_bottom_whitespace {
-        /* For visual reasons, in a message feed, we require a large
-         * bottom_whitespace to make it convenient to display new
-         * messages as they come in and prevent occluding the last
-         * message with an open compose box. Here, the bottom item
-         * is rarely interesting context for a message one is
-         * composing, but we do need at least 41px to allow the
-         * close-compose-box UI element (including border) to not
-         * overlap content. Add some more margin so that user
-         * can clearly see the end of the topics.
-         */
-        height: 120px;
-
-        #recent_view_loading_messages_indicator,
-        .bottom-messages-logo {
-            display: block;
-            position: absolute;
-            top: 200px;
-            left: 0;
-            right: 0;
-            margin: auto;
-
-            .loading_indicator_spinner {
-                position: relative;
-                top: -7px;
+    .private_conversation_row {
+        .recent_topic_stream {
+            /* Reduce padding of stream section so that user status
+                icon can have more padding without impacting height of the row */
+            padding: 5px 0 5px 8px;
+        }
+
+        .pm_status_icon {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            /* Increasing vertical padding any further will increase
+            the height of the row. */
+            padding: 8px;
+            position: relative;
+            right: -8px; /* To cancel padding-right */
+            /* To accommodate fa-group icon */
+            width: 14px;
+            height: 14px;
+
+            .fa-group,
+            .zulip-icon.zulip-icon-bot {
+                font-size: 0.8rem;
+                opacity: 0.6;
+            }
+
+            .user_circle {
+                /* Shrink the user activity circle for the Recent Conversations context. */
+                min-width: 7px;
+                height: 7px;
+                float: left;
+                position: unset;
             }
         }
     }
+}
 
-    .stream-privacy {
-        .zulip-icon {
+#recent_view_bottom_whitespace {
+    #recent_view_loading_messages_indicator,
+    .bottom-messages-logo {
+        display: block;
+        position: absolute;
+        top: 200px;
+        left: 0;
+        right: 0;
+        margin: auto;
+
+        .loading_indicator_spinner {
             position: relative;
-            left: -1px;
-            top: 1.5px;
+            top: -7px;
         }
     }
 }
 
-#recent_view {
-    display: none;
-    position: relative;
+.stream-privacy {
```

Comment:
This flattened selector is causing these styles to apply in the left sidebar, where they're defeating the layout established by CSS grid.

Reply:
Applied this while updating the deployment today.

``` diff
diff --git a/web/styles/recent_view.css b/web/styles/recent_view.css
index e0b0496412..03a12fdcbe 100644
--- a/web/styles/recent_view.css
+++ b/web/styles/recent_view.css
@@ -520,7 +520,7 @@
     }
 }
 
-.stream-privacy {
+#recent_view .stream-privacy {
     .zulip-icon {
         position: relative;
         left: -1px;
```

Reply:
Thanks! Pushed to fix it.